### PR TITLE
Add methods for process inside controller

### PR DESCRIPTION
**Pull request by ahmetanbar**

### CODE REVIEW

File: 06-Digging-Deeper/03-File-Uploads.adoc [hunk 1]
```diff
@@ -254,4 +254,42 @@ NOTE: Always make sure to call `await request.multipart.process()` to start proc
...
+If you have fields with file, you can access them with `request.multipart.field`. 
+
+[source, js]
+----
+const Drive = use('Drive')
+
+Route.post('upload', async ({ request }) => {
+
+  request.multipart.field((name, value) => {
+            body[name] = value;
```
Comment (thetutlage) [inline 06-Digging-Deeper/03-File-Uploads.adoc]: Can you please fix the indentation here? It's must be 2 spaces

Comment (thetutlage) [reply 06-Digging-Deeper/03-File-Uploads.adoc]: Also where does the `body` object comes from?

File: 06-Digging-Deeper/03-File-Uploads.adoc [hunk 2]
```diff
@@ -254,4 +254,42 @@ NOTE: Always make sure to call `await request.multipart.process()` to start proc
...
+The multipart methods are chainable. So you can chain them like 
+
+[source, js]
+----
+const Drive = use('Drive')
+
+Route.post('upload', async ({ request }) => {
+  request
+    .multipart
+    .field((name, value) => {
```
Comment (thetutlage) [inline 06-Digging-Deeper/03-File-Uploads.adoc]: Same here, badly formatted and where is the `body` object coming from?

**Review by thetutlage — CHANGES_REQUESTED** [review]

**Comment by thetutlage** [comment]
Your code examples are still inconsistent with what we have already have in our examples. For example: we do not use `;` at the end.

Can you please check and format it properly?

**Comment by RomainLanz** [comment]
Any update @ahmetanbar?

**Review by thetutlage — APPROVED** [review]

**Pull request closed without merging**